src/Models/Config.php

```diff
@@ -393,6 +393,17 @@ public function destroy(): bool
         return $createResult;
     }
 
+    public function getDecrypted(): array
+    {
+        $result = $this->readAll();
+        foreach (self::ENCRYPTED_KEYS as $column) {
+            if (!empty($result[$column])) {
+                $result[$column] = TwigFilters::decrypt($result[$column]);
```

Comment:
now is probably a good time to check if `decrypt` is used in twig templates and move this somewhere else. I'm thinking simply moving it to Filter is enough, no need to have a dedicated class as the Crypto api is dead simple.